### ISSUE TITLE
Release as an archetype

### ISSUE THREAD

**Issue by yan-kisen** [AdobeAtAdobe/aaa-aem-boilerplate#1]
I really like the way you're organizing the project. Any plans to release this as a standalone archetype?

**Comment by dbenge** [comment]
Possibly if there is enough requests for us to do that..  

I need to update the project so stay tuned.  I have some improvements that can be found in the single page application sample that I will be integrating soon.

**Comment by dbenge** [comment]
Until we get a +1 feature in GIT I guess comments on this issue will have to suffice for determining weight.

**Comment by yan-kisen** [comment]
I'm not sure what you mean by +1 feature..

I've adapted this structure for my own project, which was initially based on the multi-module archetype, by adding the site-content and config modules. Releasing an archetype could speed up development for sure, but at the same time adapting an existing project wasn't especially difficult.

**Comment by dbenge** [comment]
By +1 i was meaning if GitHub had a +1 button on issues.  This would help people express their interest in a feature.  

I am on the fence with archetype versus an example project structure.  If your experienced with AEM and Java dev then a archetype is probably preferred but if your newer to AEM an example project like this might be more help.  

We may end up putting both out if there is enough interest but for now I am going to stick to the example boilerplate for a while.

**Comment by yan-kisen** [comment]
Yes, I agree. It is valuable enough learning experience to read other peoples code. In most cases a generic archetype wouldn't apply to most organizations anyways and inexperienced people might end up with a bunch of unnecessary files because they don't understand what they're installing.

That being said, I do think that there's a certain value to hosting a generic archetype definition as a separate repo so that people can learn how to adapt it to their own needs, but that may be out of the scope of your project.

**Comment by dbenge** [comment]
Yea i agree it would be useful.  Good feedback.  Thanks
